src/ngraph/frontend/onnx_import/op/scatter_elements.cpp

```diff
@@ -0,0 +1,50 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "ngraph/opsets/opset3.hpp"
+#include "scatter_elements.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector scatter_elements(const Node& node)
+                {
+                    const auto data = node.get_ng_inputs().at(0);
```

Comment:
Check number of inputs and throw a nice exception if incorrect. Then use [] instead of at()

Reply:
To be consistent with all of the ONNX importer operators implementation, we will add such a check as a helper function in other PR.